update-dependencies.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o nounset -o errexit
```

Comment:
Maybe this should refuse to run if there are uncommitted changes in any sub-repo? (`repo` just skips over repos with changes.)